client/webpack.config.js

```diff
@@ -392,6 +393,37 @@ const webpackConfig = {
 		new ExtensiveLodashReplacementPlugin(),
 	].filter( Boolean ),
 	externals: [ 'keytar' ],
+	...( shouldUsePersistentCache
+		? {
+				cache: {
+					// eslint-disable-next-line inclusive-language/use-inclusive-words
+					// More info in https://github.com/webpack/changelog-v5/blob/master/guides/persistent-caching.md
```

Comment:
Nit: if we alter the `master` here to the sha of the last commit (https://github.com/webpack/changelog-v5/blob/f518964326583c74e9b78296faebdb9c32b01ea8/guides/persistent-caching.md), we'll achieve 2 things:

* This comment will always correspond to the right file in the right state when we were building this
* We won't need to disable the inclusive words rule above.

Reply:
Good point!